cmake_modules/FindProtobuf.cmake

```diff
@@ -26,6 +26,24 @@
 # PROTOBUF_EXECUTABLE: location of protoc
 
 
+find_package (Protobuf CONFIG)
+if (Protobuf_FOUND)
+    set (PROTOBUF_LIBRARY protobuf::libprotobuf)
+    set (PROTOBUF_STATIC_LIB PROTOBUF_STATIC_LIB-NOTFOUND)
+    set (PROTOC_LIBRARY protobuf::libprotoc)
+    set (PROTOC_STATIC_LIB PROTOC_STATIC_LIB-NOTFOUND)
+    get_target_property (target_type protobuf::libprotobuf TYPE)
+    if (target_type STREQUAL "STATIC_LIBRARY")
+        set(PROTOBUF_STATIC_LIB protobuf::libprotobuf)
+    endif ()
+    get_target_property (target_type protobuf::libprotoc TYPE)
+    if (target_type STREQUAL "STATIC_LIBRARY")
+        set (PROTOC_STATIC_LIB protobuf::libprotoc)
+    endif ()
+    get_target_property (PROTOBUF_INCLUDE_DIR protobuf::libprotoc INTERFACE_INCLUDE_DIRECTORIES)
+    return ()
```

Comment:
Before `return`, it'd be nice to print the variables as we did at line 93 - 102. Or will the protobuf's config print some info?

Reply:
Updated in the revised commit. Now it prints:
```
-- PROTOBUF_HOME: /path/to/libprotobuf
-- Found the Protobuf headers: /path/to/libprotobuf/include
-- Found the Protobuf library: protobuf::libprotobuf
-- Found the Protoc library: protobuf::libprotoc
-- Found the Protoc executable: protobuf::protoc
```